bot/arena_bot.py

```diff
@@ -46,7 +46,7 @@ async def on_disconnect(self) -> None:
     def load_extensions(self) -> None:
         """Iterate through all the `.py` files found int `bot/cogs` and load them as cogs."""
         for file in map(Path, glob("bot/cogs/*.py")):
-            self.load_extension(f"cogs.{file.stem}")
+            self.load_extension(f"bot.cogs.{file.stem}")
```

Comment:
The working Directory should be the bot folder

Reply:
Done